include/alpaka/exec/ExecCpuOmp2Blocks.hpp

```diff
@@ -148,56 +148,72 @@ namespace alpaka
                     throw std::runtime_error("Only one thread per block allowed in the OpenMP 2.0 block accelerator!");
                 }
 
-                // Force the environment to use the given number of threads.
-                int const ompIsDynamic(::omp_get_dynamic());
-                ::omp_set_dynamic(0);
-
-                // Execute the blocks in parallel.
-                // NOTE: Setting num_threads(number_of_cores) instead of the default thread number does not improve performance.
-                #pragma omp parallel
+                if (omp_in_parallel() != 0)
```

Comment:
Until now you always out `::` in front of global namespace function from OpenMP. I think this should be kept.

Reply:
Will do so